editor/js/libs/app.js

```diff
@@ -33,6 +33,9 @@ var APP = {
 			this.setCamera( loader.parse( json.camera ) );
 
 			events = {
+				appload: [],
```

Comment:
OK. But I think we should go for `init` and reserve `load` for a real loader indicating that a particular object is now completely loaded.